### ISSUE TITLE
Coloring is gone

### ISSUE THREAD

**Issue by bpgmr** [omsharp/BetterComments#14]
### Installed product versions
- Visual Studio: Visual Studio 2017, Version 15.2
- Better Comments: 2.0.47

Somehow the colors of comments got lost. If I change in the options the colors of e.g. "Comments - Important" or "Comment - Task", the comments in the code are still shown with the default color. Worked before, maybe related to the update to version 2.0.47?

Please advice, many thanks!

**Comment by omsharp** [comment]
I'm currently using version 2.0.47 and changing colors works fine on both 2017 and 2015.

After you change color and restart VS what is the case exactly?

1- The color in Fonts and Colors options page is reverted to the default.
or
2- The color in Fonts and Colors options page is changed but the change is not reflected in code editor.

**Comment by bpgmr** [comment]
Thanks for your answer and sorry for the delay, I was on vacation. The behavior was as described by "2", the color in Fonts and Colors options page was changed but the change was not reflected in code editor.

When I came back from vacation, I noticed a new update for VS, I did the update, and everything is now back to normal, coloring works as expected, don't know what caused the issue.

Sorry for the noise, and many thanks for providing and maintaining this extension!
